src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/EspressoPage.xaml

```diff
@@ -0,0 +1,182 @@
+                            HorizontalAlignment="Left"
+                            Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                            MinWidth="90"
+                            IsEnabled="{Binding Mode, Converter={StaticResource EspressoModeToReverseBoolConverter}}"
+                            SmallChange="1" 
+                            LargeChange="5"/>
+                </StackPanel>
+            </StackPanel>
+        </StackPanel>
+
+        <RelativePanel x:Name="SidePanel" 
+                    HorizontalAlignment="Left"
+                    Width="{StaticResource SidePanelWidth}"
+                    Grid.Column="1">
+            <StackPanel x:Name="DescriptionPanel">
+                <TextBlock x:Uid="About_Espresso"
+                           x:Name="AboutTitle"
+                           Grid.ColumnSpan="2"
+                           Style="{StaticResource SettingsGroupTitleStyle}"
+                           Margin="{StaticResource XSmallBottomMargin}"/>
+                <TextBlock x:Uid="Espresso_Description"
+                           TextWrapping="Wrap"
+                           Grid.Row="1" />
+            </StackPanel>
+
+            <Border x:Name="AboutImage"
+                    CornerRadius="4"
+                    Grid.Row="2"
+                    MaxWidth="240"
+                    HorizontalAlignment="Left"
+                    Margin="{StaticResource SmallTopBottomMargin}"
+                    RelativePanel.Below="DescriptionPanel">
+                <HyperlinkButton x:Uid="Espresso_ImageHyperlinkToDocs">
+                    <Image x:Uid="Espresso_Image" Source="ms-appx:///Assets/Modules/ColorPicker.png" />
+                </HyperlinkButton>
+            </Border>
+            <StackPanel x:Name="LinksPanel"
+                        Margin="0,1,0,0"
+                        RelativePanel.Below="AboutImage"
+                        Orientation="Vertical" >
+                <HyperlinkButton x:Uid="Espresso_ImageHyperlinkToDocs">
+                    <TextBlock x:Uid="Module_overview" />
+                </HyperlinkButton>
+                <HyperlinkButton NavigateUri="https://aka.ms/powerToysGiveFeedback">
+                    <TextBlock x:Uid="Give_Feedback" />
+                </HyperlinkButton>
+
+                <TextBlock 
+                x:Uid="AttributionTitle"
+                Style="{StaticResource SettingsGroupTitleStyle}" />
+
+                <HyperlinkButton Margin="0,-3,0,0"
+                                NavigateUri="https://github.com/microsoft/PowerToys/">
+                    <TextBlock Text="Den Delimarsky's Espresso" TextWrapping="Wrap" />
+                </HyperlinkButton>
+
+                <HyperlinkButton Margin="0,-3,0,0"
+                                NavigateUri="https://medium.com/@Niels9001/a-fluent-color-meter-for-powertoys-20407ededf0c">
+                    <TextBlock Text="Niels Laute's UX concept" TextWrapping="Wrap" />
```

Comment:
@niels9001 , @dend 
Which ux concept? Did I missed something or is this a copy-and-paste missing?

Reply:
Niels creates the setting view mock-ups that we see rendered in the Settings screen.